print-isoclns.c

```diff
@@ -3307,6 +3315,18 @@ isis_print(netdissect_options *ndo,
             ND_PRINT("\n\t      Sequence number: %u", GET_BE_U_4(tptr));
             break;
 
+        case ISIS_TLV_ROUTER_CAPABILITY:
+        if (tlen < 5)
+            break;
```

Comment:
Should not be silent in case of error, proposed update (also fix indent):
```patch
-        if (tlen < 5)
-            break;
+            if (tlen < 5) {
+                ND_PRINT(" [object length %u < 5]", tlen);
+                nd_print_invalid(ndo);
+                break;
+            }
```